src/main/java/org/jboss/xavier/integrations/jpa/service/WorkloadInventoryReportService.java

```diff
@@ -72,6 +72,10 @@
         return reportRepository.findAll(specification, pageable);
     }
 
+    public WorkloadInventoryReportModel findOneById(Long id) {
```

Comment:
I think we should try as much as we can to keep the `owner` check in place to avoid having "unsafe" method to read data.